inference-engine/ie_bridges/python/sample/hello_reshape_ssd/hello_reshape_ssd.py

```diff
@@ -0,0 +1,210 @@
+    h_new, w_new = cv2.imread(args.input).shape[:-1]
+    images = np.ndarray(shape=(n, c, h_new, w_new))
+    images_hw = []
+    for i in range(n):
+        image = cv2.imread(args.input)
+        ih, iw = image.shape[:-1]
+        images_hw.append((ih, iw))
+        log.info("File was added: ")
+        log.info("        {}".format(args.input))
+        image = image.transpose((2, 0, 1))  # Change data layout from HWC to CHW
+        images[i] = image
+    
+    net.reshape({input_name: [n, c, h_new, w_new]})
+    print(net.input_info['data'].input_data.shape)
+
+    # -----------------------------------------------------------------------------------------------------
+
+    # --------------------------- 4. Configure input & output ---------------------------------------------
+    # --------------------------- Prepare input blobs -----------------------------------------------------
+    log.info("Preparing input blobs")
+    assert (len(net.input_info.keys()) == 1 or len(
+        net.input_info.keys()) == 2), "Sample supports topologies only with 1 or 2 inputs"
+    out_blob = next(iter(net.outputs))
+    input_name, input_info_name = "", ""
+
+    for input_key in net.input_info:
+        if len(net.input_info[input_key].layout) == 4:
+            input_name = input_key
+            net.input_info[input_key].precision = 'U8'
+        elif len(net.input_info[input_key].layout) == 2:
+            input_info_name = input_key
+            net.input_info[input_key].precision = 'FP32'
+            if net.input_info[input_key].input_data.shape[1] != 3 and net.input_info[input_key].input_data.shape[1] != 6 or \
+                net.input_info[input_key].input_data.shape[0] != 1:
+                log.error('Invalid input info. Should be 3 or 6 values length.')
+
+    data = {}
+    data[input_name] = images
+
+    if input_info_name != "":
+        infos = np.ndarray(shape=(n, c), dtype=float)
+        for i in range(n):
+            infos[i, 0] = h
+            infos[i, 1] = w
+            infos[i, 2] = 1.0
+        data[input_info_name] = infos
+
+    # --------------------------- Prepare output blobs ----------------------------------------------------
+    log.info('Preparing output blobs')
+
+    output_name, output_info = "", net.outputs[next(iter(net.outputs.keys()))]
+    output_ops = {op.friendly_name : op for op in ops \
+                  if op.friendly_name in net.outputs and op.get_type_name() == "DetectionOutput"}
+    if len(output_ops) != 0:
+        output_name, output_info = output_ops.popitem()
+
+    if output_name == "":
+        log.error("Can't find a DetectionOutput layer in the topology")
+
+    output_dims = output_info.shape
+    if len(output_dims) != 4:
+        log.error("Incorrect output dimensions for SSD model")
+    max_proposal_count, object_size = output_dims[2], output_dims[3]
+
+    if object_size != 7:
+        log.error("Output item should have 7 as a last dimension")
+
+    output_info.precision = "FP32"
```

Comment:
Please add info to README or some other place, that output will be FP32

Reply:
Looks like no samples readmes ever mention input/output precision, so I suppose we should not add it here for consistency. Moreover, samples are not designed for any performance/accuracy testing, so we do not care much about precision in samples anyway.